### ISSUE TITLE
[CI] Fix code coverage display in README

### ISSUE THREAD

**Issue by Nashtare** [topos-network/ice-frost#1]
The README's coverage icon currently displays "unknown" for the code coverage report, although everything seems to be running fine on the CI, see below:

<img width="308" alt="Screenshot 2023-05-01 at 1 26 21 PM" src="https://user-images.githubusercontent.com/30937548/235445674-053616fb-f32f-4a89-bb5f-2efddfee8d13.png">

It's a relatively low-priority issue though.

**Comment by Nashtare** [comment]
@sebastiendan Just noticed with latest commits that now the CI job for code coverage fails with the latest error, 
```
[2023-05-01T11:34:44.580Z] ['error'] There was an error running the uploader: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 400 - [ErrorDetail(string='This repository has been deactivated. To resume uploading to it, please activate the repository in the codecov UI: https://app.codecov.io/github/topos-network/ice-frost/settings', code='invalid')]
```

Did you do something related?

**Comment by sebastiendan** [comment]
> 

The repo was deactivated on Codecov. It's been re-activated and the original problem was fixed by #2 (for code coverage fix) and #3 (for badge update).
